Header header
bool cliff_left
bool cliff_frontleft
bool cliff_frontright
bool cliff_right
bool light_bump_left
bool light_bump_frontleft
bool light_bump_centerleft
bool light_bump_centerright
bool light_bump_frontright
bool light_bump_right
bool wall_sensor

bool[11] sensors

uint16 cliff_left_data
uint16 cliff_frontleft_data
uint16 cliff_frontright_data
uint16 cliff_right_data
uint16 light_bump_left_data
uint16 light_bump_frontleft_data
uint16 light_bump_centerleft_data
uint16 light_bump_centerright_data
uint16 light_bump_frontright_data
uint16 light_bump_right_data
uint16 wall_sensor_data

uint16[11] data